frontend/test/__support__/sample_database_fixture.js

```diff
@@ -34,8 +36,16 @@ function aliasTablesAndFields(metadata) {
   }
 }
 
+function normalizeFields(fields) {
```

Comment:
The `metadata` object we pass to unit tests doesn't normalize things. I added this to normalize Fields specifically, but can add something more general if necessary.